packages/grid/README.md

```diff
@@ -0,0 +1,59 @@
+# Grid component for the Undercoat CSS Framework
+
+## Grid system
+
+The grid is based on 24 columns (tracks) by default. It's like 12 columns but more flexible since it can be split into 2, 3, 4 and even 6, 8 and 12 equal parts! But you can configure the number of columns following your requirements.
+
+The default grid settings are stored in CSS custom properties and are below (px units):
+```CSS
+  --grid-columns-number: 24; /* columns number */
+  --grid-columns-width-min: 13;    /* column (track) min width */
+  --grid-columns-width-max: 50;    /* column (track) max width */
+  --grid-width-max: 1920;    /* grid wrapper max width */
+  --grid-column-gap: 0;      /* columns gutter (gap) width */
+  --grid-margin-inline: 2vw; /* column grid inline margins */
+```
+
+Min width of the grid wrapper is set automatically by multiplying of `--grid-columns-number` on `--grid-columns-width-min` custom properties (variables). Max width is stored in `--grid-width-max` custom property and can be changed manually.
+
+> Don't use equal values for `--grid-columns-width-min` and `--grid-columns-width-max` custom properties because it leads to fixed column width, which isn't appropriate for the mobile first approach!
+
+## Layout templates
+
+The grid is based on CSS Grid Layout Module Level 1. The `grid-template-columns` property is used for specifying the track list for the grid columns. Use the `grid-template-areas` property to add specific page layouts over the grid columns.
+
+Columns help keep the UI svelte and template areas make UI components positioning simple as never before. Just specify page areas for placing UI components.
+
+## Subgrids
+
+Due to the fact that Subgrids section from CSS Grid Layout Module Level 2 specification has a weak browsers support, we have to forward subgrids manually in grid container items if it's necessary.
+
+Add the utility CSS class `.subgrid` to a grid item in the markup (HTML) to make it a new grid container with the same number of columns. Make sure the value of the `--grid-columns-number` custom property it this context is equal to the number of the columns which grid item takes. 
```

Comment:
" Make sure the value of the `--grid-columns-number` custom property **IN** this context? "

Reply:
Fixed in f512816a73938705f79f28b10c1b3f4599f762d9. Thx!